user.config

```diff
@@ -0,0 +1,8 @@
+params {
+    modules {
+        'bcftools_filter' {
+            args = "--soft-filter LowQual --exclude '%QUAL<25 || FORMAT/DP<5|| MAX(FORMAT/AD)/SUM(FORMAT/DP)<0.7 || MQ<30 || MQ0F>0.1' --output-type z"
+        }
+    }
+    remove_recombination = true
+}
```

Comment:
I'm not sure what is the purpose of this config and if it is really needed here

Reply:
This shows how a user can override a module

Reply:
Ok so maybe the `assets` directory would be a good place for it? It could also be linked as an example in the `usage.md` documentation section that you've added for it.